docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +25,22 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To connect to Iceberg, you need:
+
+* Iceberg data files stored in either Parquet or ORC format, as determined by
```

Comment:
This is more of a description of what the connector supports. The specification allows ORC, Parquet and Avro, but we don't yet support Avro. We default and recommend ORC since it performs significantly better in Trino.

I'm not sure that this is a "requirement" in the same way that a metastore is a requirement, since you could be using the connector to write data. There's no requirement that you have existing data to read.

Reply:
Fair enough. We moved it back up.